static/javascripts/kp_extraction_emb_test.js

```diff
@@ -0,0 +1,84 @@
+function submit(input_text, number_keyphrases) {
+	$("#last_group").append('<i class="fa fa-spinner fa-spin" id="spinner"></i>');
+  $('#input_text').val(input_text);
+  
+	url = "http://52.30.254.33:1277/api/kp";
+	data = {'inp_url' : input_text,
+	  'nbkp' : number_keyphrases
+	  };
+$.ajax({
+  url: "http://52.30.254.33:1277/api/kp",
+  type: "POST",
+  contentType: 'application/json',
+  dataType :"json",
+  data: JSON.stringify(data),
+  success: function(data) {
+        //Process the JSON
+        $("#spinner").remove();
+        var Itemlist = document.getElementById("list-group");
+
+        //Loop trough the JSON
+        data.api_result.forEach(function(element) {
+        element.relevance *= 100;
+        var getProcent= element.relevance ;
```

Comment:
I guess you meant to name this variable probability or smth like that?

Reply:
yes, I rename it to probabilty